configs/mx6cuboxi_nt_defconfig

```diff
@@ -38,6 +38,8 @@ CONFIG_PCI=n
 CONFIG_FSL_ESDHC=y
 CONFIG_PHYLIB=y
 CONFIG_SECURE_BOOT=y
+CONFIG_SPL_BOARD_PROVISION=y
```

Comment:
Do we want this turned on by default?  Won't it cause unprovisioned boards (dev boards) not to boot since it's waiting for input?  Should we add a timeout in spl_board_provision if it doesn't receive an input within a certain time period?

Reply:
I'll make sure that the option won't be set by default once this code merges into master. I'm keeping it this way in the Provisioning branch for demonstrating it to OEMs. It probably should send an error to the host if a MAC address isn't sent in time though.